ckanext/csrf_filter/plugin.py

```diff
@@ -95,6 +96,10 @@ def check_csrf():
         def set_csrf_token(response):
             """ Apply a CSRF token to all response bodies.
             """
+            if hasattr(response, 'response') \
+            and isinstance(response.response, GeneratorType):
```

Comment:
flake8 style checking prefers something like this:
```suggestion
                    and isinstance(response.response, GeneratorType):
```

Reply:
@wardi should this be double indented or single indented?

Reply:
flake8 doesn't like a single indent so I suggested the double:
```
E125 continuation line with same indent as next logical line
```